.github/CHANGELOG.md

```diff
@@ -119,6 +119,7 @@
   >>> mapping
   {0: (0, 1), 1: (0, 2), 2: (1, 0), 3: (1, 2), 4: (2, 0), 5: (2, 1)}
   ```
+  Add decomposition for qml.CSWAP 
```

Comment:
@MaldoAlberto it looks like this was added inside an existing changelog entry 🙂 

Could you move this down to it's own bullet point in the "Improvements" section? Thanks! Don't forget to also provide a link back to this PR, and to include your name in the 'Contributors" section :)

Reply:
@josh146  I did it   :+1: